packages/components/app/styles/components/stepper/indicator/task.scss

```diff
@@ -0,0 +1,63 @@
+//
+// STEPPER > INDICATOR > TASK
+//
+// properties within each class are sorted alphabetically
+//
+
+$hds-stepper-indicator-task-statuses: ( "incomplete", "progress", "processing", "complete" );
+$hds-stepper-indicator-task-size: 16px;
+
+// Determine states and corresponding styles
+$status-props: (
+    "incomplete": (
+        "color-default": var(--token-color-palette-neutral-300),
+        "color-hover": var(--token-color-palette-blue-200),
+        "color-active": var(--token-color-palette-blue-300),
+    ),
+    "progress": (
+        "color-default": var(--token-color-palette-blue-200),
+        "color-hover": var(--token-color-palette-blue-300),
+        "color-active": var(--token-color-palette-blue-400),
+    ),
+    "processing": (
+        "color-default": var(--token-color-palette-blue-200),
+    ),
+    "complete": (
+        "color-default": var(--token-color-palette-green-200),
+        "color-hover": var(--token-color-palette-green-300),
+        "color-active": var(--token-color-palette-green-400),
+    )
+);
+
+// Base styling for indicator::task
+.hds-stepper-indicator-task {
+    width: $hds-stepper-indicator-task-size;
+    height: $hds-stepper-indicator-task-size;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--token-color-foreground-strong);
```

Comment:
please sort props alphabetically (I know it's a pain; we will soon introduce linting for SCSS and this should become automatic).